### PR TITLE
Fix Travis config

### PR DESCRIPTION
**Pull request by Prototik**
Now travis successfully build a ABS: [build log](https://travis-ci.org/Prototik/ActionBarSherlock/builds/6843606)

Also you can setup auto deploying of snapshots in oss sonatype like [this](https://github.com/Prototik/HoloEverywhere/blob/master/.travis.yml).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,11 +8,12 @@ notifications:
...
-  - android update sdk --filter "$SDK" --no-ui --force
+  - sudo apt-get update -qq
+  - if [ `uname -m` = x86_64 ]; then sudo apt-get install -qq --force-yes libgd2-xpm ia32-libs ia32-libs-multiarch; fi
+  - wget http://dl.google.com/android/android-sdk_r21-linux.tgz
+  - tar -zxf android-sdk_r21-linux.tgz
+  - export ANDROID_HOME=`pwd`/android-sdk-linux
+  - export PATH=${ANDROID_HOME}/tools:${ANDROID_HOME}/platform-tools:${PATH}
+  - android update sdk --filter platform-tools,android-14 --no-ui --force
+  
+script: echo Tests skipped
```
Comment (JakeWharton) [inline .travis.yml:19]: What's this?

Comment (Prototik) [inline .travis.yml:19]: It's disable invoking `mvn test`.
I don't know why, but on this step local maven repo doesn't have installed abs, and dependencies cannot be resolved. (ie. i18n module cannot find dependency com.actionbarsherlock:library)

Comment (Prototik) [inline .travis.yml:19]: [See](https://travis-ci.org/Prototik/ActionBarSherlock/builds/6843378)

Comment (JakeWharton) [inline .travis.yml:19]: Well that's depressing. This is a good start, at least. Thanks!

**Pull request merged**